conans/test/integration/toolchains/intel/_base.py

```diff
@@ -0,0 +1,56 @@
+import os
+import platform
+import textwrap
+import unittest
+
+from conans.client.tools.intel import intel_installation_path
+from conans.errors import ConanException
+from conans.test.utils.tools import TestClient
+
+
+class BaseIntelTestCase(unittest.TestCase):
+    def setUp(self):
+        try:
+            installation_path = intel_installation_path(self.version, self.arch)
+            if not os.path.isdir(installation_path):
+                raise unittest.SkipTest("Intel C++ compiler is required")
+        except ConanException:
+            raise unittest.SkipTest("Intel C++ compiler is required")
+
+        self.t = TestClient()
+
+    @property
+    def version(self):
+        return '19.1'
+
+    @property
+    def arch(self):
+        return "x86_64"
+
+    @property
+    def settings(self):
+        the_os = {'Darwin': 'Macos'}.get(platform.system(), platform.system())
+        vars = [('compiler', 'intel'),
+                ('compiler.version', self.version),
+                ('build_type', 'Release'),
+                ('arch', self.arch),
+                ('os', the_os)]
+        if os.name == 'nt':
```

Comment:
platform.system == "Windows" is more used in our test suite

Reply:
okay, changed to `platform.system()` (although I remember this method to be unreliable, returning `posix` in MSYS/Cygwin environments)